modules/ROOT/pages/get-started.adoc

```diff
@@ -91,14 +91,14 @@ A partner profile consists of partner-specific settings such as:
  * Endpoints to use for sending and receiving messages to and from your partner 
  * The acknowledgements, validations, and control numbers for each message format that the partner uses
 
-Configure a partner profile for each of your trading partners. If you plan to use only host endpoints, you don’t need to create partner-specific endpoints.
+Configure a partner profile for each of your partners. If you plan to use only host endpoints, you don’t need to create partner-specific endpoints.
 
 For more information, see xref:configure-partner.adoc[].
 
 [[configure-third-party]]
 == Configure Third-Party Connections
 
-If your trading partners use third-party service providers such as VANs or 3PLs (third-party logistics providers) who manage the B2B connectivity with your organization on behalf of your partners, then create configurations for these connections.
+If your partners use third-party service providers such as VANs or 3PLs (third-party logistics providers) who manage the B2B connectivity with your organization on behalf of your partners, then create configurations for these connections.
```

Comment:
Kim didn't like using "[then](https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22,then%22asaconjunction)" this way.